### PR TITLE
feat: extend the functionality of the request executor in the discoverer

### PR DESCRIPTION
**Pull request by jobulcke**

### CODE REVIEW

File: docs/_ldi-extensions/ldes-discoverer.md [hunk 1]
```diff
@@ -14,10 +14,24 @@ follow, the structure can be discovered first.
...
+| enable-rate-limit | Boolean flag that enables limiting http requests when calling the server. (disabled when omitted)                                        | No       | N/A         | N/A                         | N/A                                                                                                                     |
+| rate-limit        | Limit of requests per period, which is defined below, that the http client should do (only on presence of enable-rate-limit)             | No       | 500         | 100                         | Integer                                                                                                                 |
+| rate-limit-period | Period in which the limit of requests, which is defined above, can be reached by the http client (only on presence of enable-rate-limit) | No       | PT1M        | PT1H                        | ISO 8601 Duration                                                                                                       |
```
Comment (Yalz) [inline docs/_ldi-extensions/ldes-discoverer.md:33]: In context for discoverer.. is this useful to mention here?

File: ldi-extensions/ldes-discoverer/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/discoverer/config/LdesDiscovererConfig.java [hunk 2]
```diff
@@ -5,14 +5,16 @@
...
+
 @Configuration
 @ConfigurationProperties
 public class LdesDiscovererConfig {
 	public static final String DEFAULT_RDF_FORMAT = "text/turtle";
 	private String url;
 	private String sourceFormat;
 	private String outputFormat;
-
+	private Map<String, String> requester;
```
Comment (Yalz) [inline ldi-extensions/ldes-discoverer/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/discoverer/config/LdesDiscovererConfig.java:17]: i see this here, but nowhere in docs. For me, it isn't clear what this map entails.. in context of this PR i presume request executor config?

**Review by Yalz — COMMENTED** [review]
Looking good, but I'm not sure if there is ample documentation?
Also, maybe some ITs with application.properties files to make sure a few of these major Request Executor cases are tackled?

File: ldi-extensions/ldes-discoverer/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/discoverer/config/LdesDiscovererConfig.java (hunk 2 above)
Comment (jobulcke) [reply ldi-extensions/ldes-discoverer/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/discoverer/config/LdesDiscovererConfig.java:17]: At first, I used a map, but then I decided to change this and I must have forgotten to remove this as well

**Pull request merged**